documentation/frontend/guides/deploy.md

```diff
@@ -33,3 +33,34 @@ For more information on how deployments work, please see the [general deployment
    number in the [changelogs](/changelogs/index), and then the tag to pass to
    the action is the version number prefixed with "rel-", for example
    "rel-2023.07.03.17.52.00".
+1. Add any new analytics events to Plausible:
+
+   1. Visit
+      [the "Goals" configuration page](https://plausible.io/openverse.org/settings/goals)
+      and add any new events.
+   1. If those events have new unique payload property names, those must also be
+      added
+      [to the "Custom properties" configuration](https://plausible.io/openverse.org/settings/properties).
+
+   ```{warning}
+   **Do not click the "click to add all existing properties" button!** This button
+   will cause all props to be added and therefore displayed on events, [even ones we've
+   explicitly removed](https://github.com/WordPress/openverse/blob/53c2e70171e22593027b184b599c530a47429a03/).
```

Comment:
This doesn't link to any specific file, should it?

Reply:
I meant to link the commit in generally, whoops! I'll update to the right link:

```suggestion
   explicitly removed](https://github.com/WordPress/openverse/commit/53c2e70171e22593027b184b599c530a47429a03).
```